emgapianns/management/lib/create_or_update_study.py

```diff
@@ -261,10 +282,42 @@ def _update_or_create_study_from_api_result(
 
     @staticmethod
     def _update_or_create_study(
-        emg_db, project_id, secondary_study_accession, defaults
+            emg_db, project_id, secondary_study_accession, defaults
     ):
         return emg_models.Study.objects.using(emg_db).update_or_create(
             project_id=project_id,
             secondary_accession=secondary_study_accession,
             defaults=defaults,
         )
+
+    def check_if_study_is_public(self, study_id):
+        url = (f"https://www.ebi.ac.uk/ena/portal/api/search?result=study&query=study_accession%3D{study_id}%20OR"
+               f"%20secondary_study_accession%3D{study_id}&limit=10&dataPortal=metagenome&includeMetagenomes=true&format"
+               "=json")
+        headers = {
+            "accept": "*/*"
+        }
+        response = requests.get(url, params=None, headers=headers)
+        if response.json() == []:
+            is_public = False
+        else:
+            is_public = response.json()[0]['secondary_study_accession'] == study_id
+
+        return is_public
+
+    def verify_study_ownership(self, study_id, submission_account_id):
+        url = f"{'https://www.ebi.ac.uk/ena/submit/report/studies'}/{study_id}"
+        ena_api_password = EMG_CONF['emg']['ena_api_password']
+        if not ena_api_password:
+            logging.warning("ENA API password is missing. Study ownership cannot be verified.")
+            return False
+        auth_string = f"mg-{submission_account_id}:{ena_api_password}"
+        headers = {
+            "accept": "*/*",
+            "Authorization": f"Basic {b64encode(auth_string.encode()).decode()}"
+        }
+        response = requests.get(url, params=None, headers=headers)
```

Comment:
```suggestion
        response = requests.get(url, headers=headers)
```
is "params" required?